docs/source/blog/datashuttle.md

```diff
@@ -0,0 +1,220 @@
+---
+blogpost: true
+date: 25 April 2024
+author: Joe Ziminski, Niko Sirmpilatze
+location: London, UK
+category: Blog
+language: English
+image: 1
+---
+
+
+
+# Managing neuroscience projects with **datashuttle**
+*Create, validate and transfer standardised project folders*
+
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-dark.png
+:align: center
+:class: only-dark
+:width: 650px
+```
+```{image} /_static/blog_images/datashuttle/datashuttle-overview-light.png
+:align: center
+:class: only-light
+:width: 650px
+```
+<br>
+
+Maintaining a well-organised neuroscience project is hard. 
+
+Although everyone can appreciate the benefits of a tidy project
+folder, the practicalities of running an experiment often gets 
+in the way. Folder organisation 
+is low on the list of priorities when it
+comes to acquisition sessions spent managing complex systems 
+and experimental animals.
+
+However, the cost of small mistakes during data acquisition can be high.
+One misplaced character may mean sessions are missed by analysis 
+scripts or subject identifications duplicated.
```

Comment:
I feel there's something syntactically wrong wit this sentence, some alternatives:

1. One misplaced character may mean sessions are missed by analysis 
scripts or subject IDs are duplicated.

2. One misplaced character may lead to subjects with duplicate IDs or to sessions being skipped by analysis scripts.

Reply:
How about `subject identifiers`? I want to avoid ID if possible just to avoid abbreviations.

Reply:
identifiers is fine.